performance/server/src/main/java/org/hawkular/apm/performance/server/Service.java

```diff
@@ -35,7 +34,7 @@
 
     private String name;
 
-    private Map<String, String> calledServices = new HashMap<String, String>();
+    private Map<String, String> calledServices;
```

Comment:
Yes it is supplied in the constructor, so creating it here was obsolete, so decided to remove it.